docs/product-administration-and-configuration-guide/src/main/asciidoc/kie-server-accessing-runtime-data-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='kie-server-accessing-runtime-data-proc']
+= Accessing Runtime Data from {CENTRAL}
+
+The `Process Instances`, `Tasks`, and `Process & Task Dashboard` pages of Business Central allow you to view the runtime data of the Intelligent Process Server. These runtime pages use the credentials of the currently logged in user to load data from the Intelligent Process Server. Therefore, to be able to view the runtime data on Business Central, ensure that the following conditions are met:
```

Comment:
Aren't we calling Business Central "Decision Central"?

Reply:
This topic isn't included in the assembly. Sorry, it's just in the PR due to the modularization required on this chapter to use parts of it.